source/ice_zbgc.F90

```diff
@@ -79,6 +80,7 @@ subroutine init_zbgc
       restart_hbrine  = .false.  ! hbrine restart
       phi_snow        = p5       ! snow porosity
       bgc_flux_type   = 'Jin2006'! type of ocean-ice poston velocity ('constant')
+      dz_ocean1       = 2.3035 ! HH first layer thickness of ocean
```

Comment:
`/g/data/ik11/inputs/access-om2/input_20201022/mom_1deg/ocean_vgrid.nc` has 2.30349969863892
Also doesn't match value in `ice_zbgc_shared.F90` --- see comments there.